src-tauri/src/main.rs

```diff
@@ -72,12 +81,59 @@ async fn control_grabbing(
     Ok(())
 }
 
+/// Engages the logger.
+fn engage_logger() -> Result<log4rs::Handle, SetLoggerError> {
+    let level: log::LevelFilter = option_env!("RUST_LOG")
+        .and_then(|s| log::LevelFilter::from_str(s).ok())
+        .unwrap_or(log::LevelFilter::Warn);
+
+    let file_path = wooting_macro_backend::config::LogFilePath::file_name();
+
+    // Build a stderr logger.
+    let stderr = ConsoleAppender::builder()
+        .encoder(Box::new(PatternEncoder::new(
+            "{d(%Y-%m-%d %H:%M:%S:%f)} | {h({({l}):5.5})} | {m}{n}",
+        )))
+        .target(Target::Stderr)
+        .build();
```

Comment:
I think the main target you should be looking at is stdout, not stderr

Reply:
Fixed in b94375c